Pyrado/pyrado/logger/experiment.py

```diff
@@ -243,6 +260,38 @@ def select_by_hint(exps: Sequence[Experiment], hint: str):
     return sl
 
 
+def create_experiment_formatter(
+    show_hparams: Optional[List[str]] = None, show_extra_info: bool = True
```

Comment:
the docsting must explain every arument (:param: ... and optionally :return: ...)